assets/js/src/core/components/modal/footer/modal-footer.tsx

```diff
@@ -13,22 +13,31 @@
 
 import React from 'react'
 import { useStyle } from '@Pimcore/components/modal/footer/modal-footer.styles'
+import { Flex } from '@Pimcore/components/flex/flex'
 
 interface IModalFooterProps {
   children?: React.ReactNode
   buttonAlignment?: 'space-between' | 'end'
```

Comment:
I think we can improve this a bit more. I think we used that before we had our flex component.
We could define the justify prop from flex as prop here.

Reply:
Ah! I was meant to be using the shared component flex container! 
Good spot :)

Reply:
@vin0401 fixed :)